src/thor/timedep_reverse.cc

```diff
@@ -511,7 +513,8 @@ void TimeDepReverse::SetOrigin(GraphReader& graphreader,
     // DO NOT SET EdgeStatus - it messes up trivial paths with oneways
     uint32_t idx = edgelabels_rev_.size();
     edgelabels_rev_.emplace_back(kInvalidLabel, opp_edge_id, edgeid, opp_dir_edge, cost, sortcost,
-                                 dist, mode_, c, false, -1);
+                                 dist, mode_, c, false, !(costing_->IsClosed(opp_dir_edge, tile)),
```

Comment:
im pretty sure this should be `directededge` from above. its the state of the destination edge the route will actually end on that we need to use. `opp_dir_edge` is the opposing edge which we use to carry out the backwards search.

Reply:
Ah, yes. the directed edge is create just a few lines above. I didn't see it when doing copy-pasta!